README.md

```diff
@@ -1,37 +1,60 @@
-3. Install Node.js. We're currently using Node.js version 14, the long term support (LTS) release. You can install the latest version 14 LTS release at [nodejs.org](https://nodejs.org/en/download/).
-    - You can verify your node version by running `node -v` in the project directory.
-    - If you want a simpler way to manage multiple versions of node, you can use Node Version Manager (AKA nvm) to install the version docs requires. Follow the directions [here](https://github.com/nvm-sh/nvm#installing-and-updating) to install NVM. Then run `nvm install` in the project directory, followed by `nvm use` to use the version specified in the project's `.nvmrc` file.
-4. Ensure you have Python 3.6 or higher by running `python3 -V`. This should return a version number. If your version is less than 3.6, or you get an error, install Python with (MacOS) `brew install python3`. Python is not needed for the core Gatsby system, but is required by several source scripts.
-5. Install yarn and gatsby with `npm i -g gatsby-cli` and `npm i -g yarn`. Gatsby is the software that runs the docs site, and Yarn is an alternative package manager which will replace `npm` for this project.
-6. Install all required packages by simply running `yarn`.
-7. Pull the shared icon files down with `git submodule update --init`
-8. Select sources with `yarn config-sources`, running `yarn pull-sources` as part of this process if prompted (see section below for details)
-9. Run the site locally with `yarn develop`. The site should now be running at `http://localhost:8000/`!
+## MacOS Installation
+We recommend using MacOS to work with the EDB Docs application.
 
-## Running Without a Local Installation (Recommended for Windows)
+1. Install [Homebrew](https://brew.sh/), if it's not already installed. (Use `brew -v` to check.)
 
-If you wish to work with Docs without installing the prerequesites locally, you can do so from within a Docker container using VSCode. See: [Working on Docs in a Docker container using VSCode](README_DOCKER_VSCODE.md)
+1. Install Git using Homebrew with `brew install git`, if it's not already installed.  (Use `git --version` to check.)
+
+1. Clone the [EDB Docs GitHub repo](https://github.com/EnterpriseDB/docs) to your local machine using [GitHub Desktop](https://desktop.github.com/), or via the Terminal with `git clone https://github.com/EnterpriseDB/docs.git`
+
+1. Set up an SSH key in GitHub, if you haven't done so already. (Go to [GitHub's SSH Keys page](https://github.com/settings/keys) to check.) If you don't have an SSH Key set up yet, you'll need to set one up to authenticate you to GitHub. See [GitHub's SSH docs](https://docs.github.com/en/github-ae@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) for more information.
```

Comment:
If this is going to be a full number item, I think it should be above the `clone` step, and then the clone step should use `git clone git@github.com:EnterpriseDB/docs.git`, which would only work if they set their SSH key up correctly